examples/multi_skill_bot/data/GreetBot/domain.yml

```diff
@@ -0,0 +1,15 @@
+intents:
+  - greet
+  - goodbye
+
+actions:
+- utter_greet
+- utter_goodbye
+
+templates:
+
+  utter_did_that_help:
+  - text: "Did that help you?"
+
+  utter_happy:
+  - text: "Great carry on!"
```

Comment:
Templates for greet and goodbye are missing.
```
  utter_greet:
  - text: "Hello!"

  utter_goodbye:
  - text: "Bye!"
```

Reply:
good catch, fixed